apps/server/prisma/seed.ts

```diff
@@ -0,0 +1,188 @@
+import { PrismaClient } from '@prisma/client';
+import { v4 as uuidv4 } from 'uuid';
+
+const prisma = new PrismaClient();
+
+const LEADERSHIP_TEAM_UUID = uuidv4();
+const CHIEF_OF_STAFF_UUID = uuidv4();
+const CHIEF_FIN_OFFICER_UUID = uuidv4();
+const AGG_DIR_UUID = uuidv4();
+const CHIEF_LEARNING_ENGAGEMENT_UUID = uuidv4();
+
+async function main() {
+
+  const formTemplate1Id = uuidv4();
+  const formTemplate1 = await prisma.formTemplate.upsert({
+    where: { id: formTemplate1Id },
+    update: {},
+    create: {
+      id: formTemplate1Id,
+      name: 'Form Template 1',
+      formDocLink: 'https://www.mfa.org/'
+    },
+  });
+
+  const signatureFields = await fetchSignatureFields(formTemplate1Id);
+
+  const departmentLeadershipTeam = await prisma.department.upsert({
+    where: { id: LEADERSHIP_TEAM_UUID },
+    update: {},
+    create: {
+      id: LEADERSHIP_TEAM_UUID,
+      name: 'Leadership Team',
+    },
+  });
+  const positions = [
+    {
+      id: CHIEF_OF_STAFF_UUID,
+      name: 'Chief of Staff',
+      departmentId: departmentLeadershipTeam.id,
+      signatureFields: signatureFields,
+    },
+    {
+      id: CHIEF_FIN_OFFICER_UUID,
+      name: 'Chief Financial Officer',
+      departmentId: departmentLeadershipTeam.id,
+      signatureFields: [],  // assuming no signature fields for this position
+    },
+    {
+      id: AGG_DIR_UUID,
+      name: 'AGG Director',
+      departmentId: departmentLeadershipTeam.id,
+      signatureFields: [],  // assuming no signature fields for this position
+    },
+    {
+      id: CHIEF_LEARNING_ENGAGEMENT_UUID,
+      name: 'Chief of Learning & Community Engagement',
+      departmentId: departmentLeadershipTeam.id,
+      signatureFields: [],  // assuming no signature fields for this position
+    },
+  ];
+  for (const position of positions) {
```

Comment:
we can probably make separate functions for upserting positions, employees, etc, just so we can keep the main function cleaner?